packages/frontend/src/components/PageSpinner/index.tsx

```diff
@@ -1,10 +1,10 @@
-import { Spinner } from '@blueprintjs/core';
-import React, { FC } from 'react';
+import { Loader } from '@mantine/core';
+import { FC } from 'react';
 import { LogoContainer, SpinnerWrapper } from './PageSpinner.styles';
 
 const PageSpinner: FC = () => (
     <SpinnerWrapper data-testid="page-spinner">
-        <Spinner size={100} />
+        <Loader color="gray.5" size={100} />
```

Comment:
Let me know what you think @IrakliJani (cc @PriPatel )

See here example:

https://github.com/lightdash/lightdash/assets/7611706/55b3d6a6-334e-48cc-a67c-3a1d491927c7

Reply:
@joaoviana can we make it thinner. like.. way thinner?

Reply:
IDK how! 🤣
checking!

Reply:
Yeah it looks a bit thic

Reply:
https://github.com/lightdash/lightdash/assets/7611706/8ed6e0a6-e99e-46c5-a6c5-87aa5dca5533

not thicc anymore!